servers/rendering_server.cpp

```diff
@@ -2346,6 +2346,9 @@ RenderingServer::RenderingServer() {
 	GLOBAL_DEF("rendering/quality/depth_prepass/enable", true);
 	GLOBAL_DEF("rendering/quality/depth_prepass/disable_for_vendors", "PowerVR,Mali,Adreno,Apple");
 
+	GLOBAL_DEF("rendering/quality/texture_filters/mipmap_lod_bias", 0.0);
+	// The highest supported value is currently `16` according to <https://opengl.gpuinfo.org/displaycapability.php?name=GL_MAX_TEXTURE_LOD_BIAS>.
```

Comment:
Since this applies to Vulkan and not OpenGL, we should cite the Vulkan limits

https://vulkan.gpuinfo.org/displaydevicelimit.php?name=maxSamplerLodBias

Reply:
Actually, just remove the comment altogether. A bias of 16 on a 65536x65536 texture would read from the 1x1 mipmap. We really don't need to worry about people needing larger biases than that.